README.md

```diff
@@ -6,7 +6,7 @@ We have moved TorchDynamo to
 - `import torchdynamo` is now `import torch._dynamo`
 - `import torchinductor` is now `import torch._inductor`
 
-For Documentation: https://pytorch.org/docs/stable/dynamo/index.html
+For Documentation: https://pytorch.org/docs/stable/torch.compiler.html
```

Comment:
This link seems broken too? 

https://pytorch.org/docs/stable/torch.compiler.html￼

Reply:
Ah nevermind, some training unicode it seems.